editor/src/components/inspector/sections/layout-section/flex-element-subsection/flex-element-subsection.tsx

```diff
@@ -65,59 +72,96 @@ export const FlexElementSubsectionExperiment = betterReactMemo(
+}
 
-    const isMainAxisVisible =
-      isMainAxisVisibleForFlexDirection ||
-      isNotUnsetDefaultOrDetected(alignSelf.controlStatus) ||
-      isNotUnsetDefaultOrDetected(flexBasis.controlStatus) ||
-      isNotUnsetDefaultOrDetected(flexGrow.controlStatus) ||
-      isNotUnsetDefaultOrDetected(flexShrink.controlStatus)
+export function useInitialAdvancedSectionState(): boolean {
+  const alignSelf = useInspectorLayoutInfo('alignSelf')
+  return isNotUnsetDefaultOrDetected(alignSelf.controlStatus)
+}
 
-    const [mainAxisControlsOpen, setMainAxisControlsOpen] = usePropControlledStateV2(
-      isMainAxisVisible,
+export function useInitialSizeSectionState(): boolean {
+  const flexBasis = useInspectorLayoutInfo('flexBasis')
+  const flexGrow = useInspectorLayoutInfo('flexGrow')
+  const flexShrink = useInspectorLayoutInfo('flexShrink')
+  return (
+    isNotUnsetDefaultOrDetected(flexBasis.controlStatus) ||
+    isNotUnsetDefaultOrDetected(flexGrow.controlStatus) ||
+    isNotUnsetDefaultOrDetected(flexShrink.controlStatus)
+  )
+}
+
+const MainAxisControls = betterReactMemo(
+  'MainAxisControls',
+  (props: FlexElementSubsectionProps) => {
+    const colorTheme = useColorTheme()
+
+    const initialIsFixedSectionVisible = useInitialFixedSectionState(props.parentFlexDirection)
+    const initialIsAdvancedSectionVisible = useInitialAdvancedSectionState()
+
+    const [fixedControlsOpen, setFixedControlsOpen] = usePropControlledStateV2(
+      initialIsFixedSectionVisible,
     )
-    const toggleSection = React.useCallback(() => setMainAxisControlsOpen(!mainAxisControlsOpen), [
-      mainAxisControlsOpen,
-      setMainAxisControlsOpen,
+    const toggleFixedSection = React.useCallback(() => setFixedControlsOpen(!fixedControlsOpen), [
+      fixedControlsOpen,
+      setFixedControlsOpen,
     ])
+    const [advancedControlsOpen, setAdvancedControlsOpen] = usePropControlledStateV2(
+      initialIsAdvancedSectionVisible,
+    )
+    const toggleAdvancedSection = React.useCallback(
+      () => setAdvancedControlsOpen(!advancedControlsOpen),
+      [advancedControlsOpen, setAdvancedControlsOpen],
+    )
     return (
       <>
         <InspectorSubsectionHeader style={{ display: 'flex', justifyContent: 'space-between' }}>
           <span>Main Axis</span>
-          <SquareButton highlight onClick={toggleSection}>
-            <ExpandableIndicator
-              testId='flex-element-subsection'
-              visible
-              collapsed={!mainAxisControlsOpen}
-              selected={false}
-            />
-          </SquareButton>
+          <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+            <Button highlight onClick={toggleFixedSection} primary={fixedControlsOpen}>
+              <InlineLink
+                style={{
+                  color: fixedControlsOpen ? colorTheme.white.value : colorTheme.primary.value,
+                  opacity: !initialIsFixedSectionVisible && !fixedControlsOpen ? 0.3 : 1,
```

Comment:
Can you flip the condition here please? `initialIsFixedSectionVisible || fixedControlsOpen ? 1 : 0.3`. And likewise below